rfcs/ambassador-program/index.md

```diff
@@ -0,0 +1,83 @@
+* Social proof - named individuals with high visibility (and a proportion from known corporate brands) as Ambassadors
+* Increased GraphQL content online and off
+* Community building - increase the number of “known faces” within GraphQL, increased approachability (addressing the concern of “lowering the barrier to join an inclusive GraphQL community” raised by the Community WG)  
+
+How can these goals be measured? 
+
+* Engagement metrics - social, chat channels, search, downloads, page views…
+* New community members (across social media comments, Discord users, local gathering attendees, working group members…) 
+* Volume of talks at conferences
+* Volume of articles written
+
+### Ambassador Tasks
+
+As a GraphQL Ambassador, you will be an important touchstone for people interested in GraphQL to learn more about it in a friendly and constructive way. 
+
+One or more of the following:
+
+* Conference & meetup talks on GraphQL
+* Local gathering organization
+* Written GraphQL content
+* Video / audio GraphQL content
+
+### Foundation Tasks
+
+* Grant available for travel expenses for approved conference & meetup talks
+* Microgrant (or honorarium) available for content creation 
+    * Where is this content hosted? 
+* Promotion of Ambassador material - conference talk videos and created content
+* Promotion of the Ambassador as a go-to GraphQL expert
+    * Website profile
+    * Longer task: biographies / interviews with each Ambassador
+    * Credly badge
+* Exclusive GraphQL swag eg Ambassador hoodie
+
+### Ambassador Requirements
+
+* Highly knowledgeable about GraphQL and readily able to articulate those ideas 
+* Technical in tone, without sounding like they are on a sales team
+* Be a model community member: Adhering to the code of conduct especially when critical of GraphQL or defending it to others criticism, remaining respectful, constructive and fair 
+* Do not come across to other community members as selling another product, service or agenda
+
+
+### Ambassador Code of Conduct
+
+While the wider GraphQL community abides by the [contributor covenant code of conduct](https://graphql.org/codeofconduct/), we will need a stronger set of standards for the Ambassador Program. This is because the Foundation will be endorsing these people as representatives. We can look to the [CNCF Ambassador Program Standards of Excellence](https://www.cncf.io/people/ambassadors/program-standards/) as a good indicator of what this should be. In fact, we are encouraged to borrow liberally from them, as they are also a Linux Foundation project with an already successful program, and their lawyers have already signed it off. 
+
+### Process
+
+Yearly program
+ * 1 year term? Or mirror the TSC and use a 2 year term  
+
+How many Ambassadors is a reasonable number?  
+
+Self-nominations and nominations from within the community; who should review these?  
+
+Process for removing an Ambassador? And the reason to do so?  
```

Comment:
Probably something about prolonged inactivity? I wouldn't necessarily put too much process there and leave it at the discretion of the "pastors" (or however we decide to call the group that is going to supervise the program)

Reply:
The problem is you need a concrete process before ever having to use it. In running events and community spaces before, I've found it much easier to say "what does the code of conduct say?" instead of thinking on my feet. Having a process in place also means you are less likely to be accused of unfairness if an incident does happen. However, we also have the backstop of the legal team at Linux Foundation to help us here.

Reply:
Agreed but we don't want to make it too administrative as well. I would try to word it as generally as possible: "we expect you to do X contributions/year" (contributions probably being voluntarily open as some will be bigger blog post authors, others conference talks, others YT videos, etc...). 

The dress code analogy works well. We want to give guidance but still leave some room for improvisation/creativity.